app/controllers/application_records_controller.rb

```diff
@@ -132,4 +130,13 @@ def given_or_all_department_ids
   def record_params
     params.permit(:position_id, :ethnicity, :gender)
   end
+
+  def modify_params
+    parameters = {}
+    parameters[:date] = if params[:date_for_later].present?
+                          Date.strptime(params[:date_for_later], '%m/%d/%Y')
+                        end
+    parameters[:mail] = true if params[:mail_to_applicant] == '1'
```

Comment:
This line can be shortened to `parameters[:mail] = params[:mail_to_applicant] == 1`